<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthenticationDetails</class>
 <widget class="QWidget" name="AuthenticationDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_gbAuthentication">
     <property name="toolTip">
      <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
     </property>
     <property name="title">
      <string>Requires HTTP authentication</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>m_txtUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>m_txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
